diff -ru1 nltk-2.0.5-orig/distribute_setup.py nltk-2.0.5/distribute_setup.py
--- nltk-2.0.5-orig/distribute_setup.py	2015-01-12 17:44:22.000000000 -0500
+++ nltk-2.0.5/distribute_setup.py	2018-06-07 12:05:26.986687392 -0400
@@ -49,3 +49,3 @@
 DEFAULT_VERSION = "0.6.21"
-DEFAULT_URL = "http://pypi.python.org/packages/source/d/distribute/"
+DEFAULT_URL = "https://pypi.python.org/packages/source/d/distribute/"
 SETUPTOOLS_FAKED_VERSION = "0.6c11"
